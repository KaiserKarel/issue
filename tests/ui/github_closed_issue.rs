

#[issue::track(url="https://github.com/KaiserKarel/issue/issues/2")]
fn main() {

}