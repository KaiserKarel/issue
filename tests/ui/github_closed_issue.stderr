error: issue https://github.com/KaiserKarel/issue/issues/2 has been closed
 --> $DIR/github_closed_issue.rs:3:16
  |
3 | #[issue::track(url="https://github.com/KaiserKarel/issue/issues/2")]
  |                ^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/github_closed_issue.rs:3:1
  |
3 | / #[issue::track(url="https://github.com/KaiserKarel/issue/issues/2")]
4 | | fn main() {
5 | |
6 | | }
  | |_^ consider adding a `main` function to `$DIR/ui/github_closed_issue.rs`
